var/spack/repos/builtin/packages/lcc/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Lcc(CMakePackage):
+    """Los Alamos Crystal Cut (LCC) is a simple crystal builder. It is an easy-to-use
+    and easy-to-develop code to make crystal solid/shape and slabs from a crystal
+    lattice. Provided you have a ‘.pdb‘ file containing your lattice basis you can
+    create a solid or slab from command line."""
+
+    homepage = "https://github.com/lanl/LCC.git"
+    url = "https://github.com/lanl/LCC/archive/refs/tags/v1.0.1.tar.gz"
+    git = "https://github.com/lanl/LCC.git"
+
+    maintainers("cnegre")
+
+    version("main", branch="main", preferred=True)
+    version("1.0.1", sha256="fa13364dcdf3b1f8d80fc768f0e7ad3849f8d98091fb96926100a6764f836020")
+    version("1.0.0", sha256="750ce09e809a4e85ae3219fd537dc84a923fe3d3683b26b5d915eccfd1f0120c")
```

Comment:
Also it's not needed for this PR, but you should make stable tarball for these (and call `v1.0.1` `v1.0.1`, not `cmake` as it says in your releases).  See https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/